modules/assim.sequential/inst/covariates.R

```diff
@@ -0,0 +1,109 @@
+##' @title Covariate Data Prep
+##' @author Joshua Ploshay
+##'
+##' @description This script lists the code needed to download, resample, and 
+##' stack the covariate data used in the downscale_function.R sript.
+##' 
+##' @return A spatraster object consisting of a stack of maps with labeled layers
+
+#### WorldClim ####
+# WorldClim v2.1 (1970-2000) historical climate data
+# Data can be found at https://www.worldclim.org/data/worldclim21.html
+# Product is a zip file containing 12 .tif files, one for each month
+# Use code below to average the 12 files to obtain one map
+# 2023 REU project used 10 minute spatial resolution
+
+## Solar Radiation (kJ m-2 day-1)
+srad <- terra::rast(list.files(path = "/projectnb/dietzelab/jploshay/pecan_copy/jploshay/10m_srad",
```

Comment:
For anyone to be able to reuse this script they will have to redefine a LOT of local paths. Better would be to define ONE folder where all the data is located at the top of the script and then express all the other paths relative to that (e.g. using `file.path()`)